features/zones/Create.feature

```diff
@@ -68,5 +68,6 @@ Feature: The create Zone flow works
         zoneInsight:
           subscriptions:
             - connectTime: '2020-07-28T16:18:09.743141Z'
+              status: {}
```

Comment:
How comes we added this here? Is `status` always an empty object immediately on zone creation? I also noticed we do this through this PR, does doing that have any consequences for what we are covering in the test?

Reply:
There some pages that access this object (via `SubscriptionDetails`) and setting this as an empty object is the most minimal way of getting the errors out of those tests.

Reply:
So when you say "getting the errors out of those tests" do you mean "getting errors out" as in making the thing error, or making sure it doesn't error? I think its the latter but just checking

Reply:
Making sure the application code doesn't throw an exception on account of accessing `status.property` (as `status` would be wholly absent from the mock data without this change).

Reply:
Hmm I might catch up with you elsewhere on this one, just so I follow completely.